### DISCUSSION TITLE
downloading user overview using formr package

### DISCUSSION THREAD

**Discussion by fidadoma** [rubenarslan/formr.org#559] [Q&A] [answered]
Hi all,
I am preparing overview scripts for our study, and I am unsure whether I could somehow download user overviews (or run data in long format ) by script. Individual surveys are fine, but I would need to see a history of participant movements throughout the study (we have a complicated diary study). Can it be done (I really hope that I do not to press download user overview data every day manually :-) 
thanks
Filip

**Comment by cyriltata** [comment]
Hi Filip, you can do this by getting your data over the formr API. I will
need to generate credentials for you if you are using our instance

**Reply by fidadoma** [reply]
Hi Cyril we are using our own instance (personally maintained by Ruben:)), which function in formr package is that please?

**Reply by rubenarslan** [reply, marked as answer]
Hi @fidadoma I just added two functions to download user overview/detail to the formr R package. https://github.com/rubenarslan/formr `formr_user_overview("run_name", host = yourinstance)` `formr_user_detail("run_name", host = yourinstance)` 
Hope this helps.

**Reply by fidadoma** [reply]
wow, thank you! that was it:-)
